packages/curvenote/src/sync/init.ts

```diff
@@ -150,27 +150,11 @@ export async function init(session: ISession, opts: Options) {
   }
   // If there is a new project config, save to the state and write to disk
   if (projectConfig) writeConfigs(session, path, { projectConfig });
-  const state = session.store.getState();
   // Personalize the config
   session.log.info(`📓 Creating site config`);
   me = await me;
   siteConfig.title = title;
   siteConfig.logo_text = title;
-  siteConfig.nav = (siteConfig.projects || [])
-    .map((proj) => {
-      if (proj.path) {
-        const projConf = selectors.selectLocalProjectConfig(state, proj.path);
-        return {
-          title: projConf?.title || proj.slug,
-          url: `/${proj.slug}`,
-        };
-      } else if (proj.remote) {
-        return { title: proj.slug, url: proj.remote };
-      } else {
-        return undefined;
-      }
-    })
-    .filter((proj): proj is SiteNavPage => Boolean(proj));
```

Comment:
@rowanc1 so now nav creation will happen when? on load in the theme?

Reply:
This is just the init function for creating the default nav. The main way it is used isn't that helpful ... It just duplicates the title. Removing that means the defaults are visually better, but you have to go to the docs to learn how to put a nav item in.